ui/client/components/AddProcessForm.tsx

```diff
@@ -50,7 +51,7 @@ export function AddProcessForm({nameValidators, value, onChange}: AddProcessForm
                 value={value.processId}
                 onChange={e => onFieldChange("processId", e.target.value)}
               />
-              <ValidationLabels validators={nameValidators} values={[value.processId]}/>
+              <ValidationLabels validators={nameValidators} values={[value.processId]} processNameValidationError={processNameValidationError}/>
```

Comment:
What about scenario name change? How do we handle it?

Reply:
We shouldn't pass this specific error to generic ValidationLabels. Let's add custom validator instead